src/game.h

```diff
@@ -166,6 +167,7 @@ class game
         friend scent_map &get_scent();
         friend timed_event_manager &get_timed_events();
         friend memorial_logger &get_memorial();
+        friend void rummage_pocket_activity_actor::finish(player_activity&, Character&);
```

Comment:
Why does this need to be a friend? I do not think it should be.

Reply:
It's because `finish` call `game::wield` so without that it wouldn't work ( at least that's what rsulii55 had explained in his PR)

Reply:
Okay, that being in game (in any form - there are npc:: and avatar:: versions, but nothing should be in game at all) is entirely insane, but not your fault.

Reply:
so there's avatar::wield? If so I can change it to use that instead

Reply:
There's an avatar::wield, but it looks like a lot of sanity checking happens in `game::wield`.